docs/conf.py

```diff
@@ -65,6 +70,46 @@
 # (note that `group` is the GitHub user or organization)
 issues_github_path = "skops-dev/skops"
 
+
+def linkcode_resolve(domain, info):
+    if domain != "py":
+        return None
+    if not info["module"]:
+        return None
+    filename = info["module"].replace(".", "/")
+    if domain not in ("py", "pyx"):
+        return
```

Comment:
this is not needed, we have already checked for `py` domain above.

Reply:
✅ Done